### PR TITLE
Add test data for Shufflenet ONNX

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

**Comment by dkurt** [comment]
@l-bat, please check [diffs](https://github.com/opencv/opencv_extra/pull/523/files) page. Does all the filed need to be changed?

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 1]
```diff
@@ -200,3 +202,19 @@ def forward(self, x):
...
+        self.squeeze1 = nn.Linear(3, 4, bias=True)
+
+    def forward(self, x):
+        x = self.squeeze1(x)
+        return torch.t(x)
+
+
+input = Variable(torch.randn(2, 3))
+model = Transpose()
+save_data_and_model("transpose", input, model)
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py]: @l-bat, looks like we need to add this test into `test_onnx_importer.cpp`.

Comment (l-bat) [reply testdata/dnn/onnx/generate_onnx_models.py]: done

**Comment by dkurt** [comment]
Please consider https://github.com/opencv/opencv_extra/pull/499#pullrequestreview-158868154 in this PR.

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 2]
```diff
@@ -200,3 +202,19 @@ def forward(self, x):
                 )
 input = Variable(torch.randn(1, 3, 32, 32))
 save_data_and_model("constant", input, model)
+
+
+class Transpose(nn.Module):
+
+    def __init__(self):
+        super(Transpose, self).__init__()
+        self.squeeze1 = nn.Linear(3, 4, bias=True)
```
[the remark this answers is not in the document]
Comment (l-bat) [reply testdata/dnn/onnx/generate_onnx_models.py]: done
